src/Drivers/Typo3CmsDriver.php

```diff
@@ -16,8 +16,13 @@ public function getDatabaseConfig(string $databaseConfiguration = 'Default'): ar
         exec($this->getPhpExecCommand() . ' ./vendor/bin/typo3cms configuration:showactive DB --json', $output,
             $resultCode);
         if ($resultCode === 0) {
-            return json_decode(implode("\n", $output), true, 512,
+            $config = json_decode(implode("\n", $output), true, 512,
```

Comment:
Is this only about "flags", right? So could be also:

```
             $config = json_decode(implode("\n", $output), true, 512,
                JSON_THROW_ON_ERROR)['Connections'][$databaseConfiguration];
            $config['flags'] = $config['driverOptions']['flags'] ?? 0;
            return $config;
```

Reply:
Yes, you're right. I was unsure if there could be other options in arrays - but I just re-checked and couldn't find any. So your line will be enough - I'll do a quick update.